src/scene/scene_lua.inl

```diff
@@ -547,7 +551,7 @@ namespace {
     std::vector<std::string> res;
     for (properties::Property* prop : props) {
         // Check the regular expression for all properties
-        const std::string& id = prop->uri();
+        const std::string_view id = prop->uri();
```

Comment:
Same thing here, could we rename the variable to make it clearer that we're working with the full uri and not the identifier?
```suggestion
        const std::string_view uri = prop->uri();
```